source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -502,3 +606,63 @@ void ListParser::CaptureListToken()
     m_parsedResult.Clear();
     m_parsedResult.AppendToTokens(codeGen);
 }
+
+// list has form of -\s+ markdown block and a line break
```

Comment:
>- [](start = 20, length = 1)

For an ordered list this is a digit, right?

Reply:
thank you, you are correct.

---
In reply to: [158143255](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158143255) [](ancestors = 158143255)